.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: java
 
+os:
+  - linux
+  - osx
```

Comment:
Cool that we test on an additional OS now! :+1:

Reply:
I think they working on windows support too but it is not ready as you can see in this [build](https://travis-ci.org/junit-pioneer/junit-pioneer/builds/652600148) [Windows OpenJDK 11](https://travis-ci.org/junit-pioneer/junit-pioneer/jobs/652600151)